avoid duplicated Content-transfer-encoding.
the patch is suppolied from sendmail.org.


*** mime.c.orig	Tue Aug 11 14:29:13 1998
--- mime.c	Sun Aug 16 18:17:44 1998
***************
*** 444,451 ****
  	if (sectionhighbits == 0)
  	{
  		/* no encoding necessary */
! 		if (cte != NULL)
  		{
  			snprintf(buf, sizeof buf,
  				"Content-Transfer-Encoding: %.200s", cte);
  			putline(buf, mci);
--- 444,455 ----
  	if (sectionhighbits == 0)
  	{
  		/* no encoding necessary */
! 		if (cte != NULL && bitset(MCIF_INMIME, mci->mci_flags))
  		{
+ 			/*
+ 			** Skip _unless_ in MIME mode; see putheader() for the
+ 			** counterpart where this is skipped _if_ in MIME mode.
+ 			*/
  			snprintf(buf, sizeof buf,
  				"Content-Transfer-Encoding: %.200s", cte);
  			putline(buf, mci);
